hooks/gen_schema.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+ROOT_DIR=$(realpath ${0%/*/*} --canonicalize-existing)
```

Comment:
the version of `realpath` that ships with macOS by default also doesn't have the `--canonicalize-existing` flag. is it necessary?

Reply:
We can do away with it.